### PR TITLE
Use mockito spy to verify if method is being called or not, this removes all Thread.sleep calls.

### PR DESCRIPTION
**Pull request by pavolloffay**
cc @jpkrohling

### CODE REVIEW

File: opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/TracingFilterTest.java [hunk 1]
```diff
@@ -272,10 +276,9 @@ public void testExcludePattern() throws IOException, InterruptedException {
                     .build();
 
             client.newCall(request).execute();
-            // static wait to make sure span is not created
-            Thread.sleep(50);
```
Comment (jpkrohling) [inline opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/TracingFilterTest.java:276]: Did I miss this? O_O
I'm getting old.

**Review by jpkrohling — APPROVED** [review]
LGTM

File: opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/TracingFilterTest.java (hunk 1 above)
Comment (pavolloffay) [reply opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/TracingFilterTest.java:276]: (~~I was happy that you missed it~~)

**Comment by pavolloffay** [comment]
@jpkrohling thanks!

**Pull request merged**